static/styles/themes/dark-theme.scss

```diff
@@ -458,18 +458,21 @@ textarea.form-control {
 }
 
 .opt-decoration {
-    .analysis,
-    .mixed {
-        background: #fdfd96 !important;
-    }
+    width: 8px !important;
```

Comment:
I think these rules are redundant as they're in `explorer.scss` (which applies always regardless of theme)

Reply:
Yup, specially layout code should be in `explorer.scss` file